### DISCUSSION TITLE
Bitnami Docker - Email Domains issue

### DISCUSSION THREAD

**Discussion by cloudcogsio** [mastodon/mastodon#22459] [Q&A] [answered]
I have an instance running on Docker using the Bitnami image v4.0.2 using docker-compose.

The instance will not accept any signups by showing "uses a disallowed e-mail provider" beneath the email field when an attempt is made to process the signup.

Environment variables
EMAIL_DOMAIN_ALLOWLIST and EMAIL_DOMAIN_DENYLIST are NOT set.

Registration->Who can sign-up is set to "Anyone can signup"

Any help/suggestions appreciated.

**Comment by cloudcogsio** [comment, marked as answer]
After investigating it seems the bitnami docker image is setting an empty EMAIL_DOMAIN_ALLOWLIST environment variable.
Modifying /opt/bitnami/scripts/mastodon-env.sh to remove the variable export resolved the issue.

**Reply by natehouk** [reply]
Thank you @cloudcogsio, I ran into this same issue.
